src/test/java/com/ericsson/gerrit/plugins/eiffel/events/generators/EiffelEventGeneratorTest.java

```diff
@@ -110,6 +112,13 @@ public void testEiffelEventGeneratorSshBaseUrlException() {
         assertEquals("Repo URI should have been set to null", null, repoURI);
     }
 
+    @Test
+    public void testEiffelEventGeneratorNullHost() {
+        String repoURI = EiffelEventGenerator.createRepoURI(NULL_HOST, PROJECT);
+
+        assertNull("Repo URI should have been set to null", repoURI);
```

Comment:
Maybe not part of this story but do we really want to use null?

Reply:
In this case I believe so. When you serialize the data model the values that are null will be ignored. We assign one of the values in the model to whatever is returned by createRepoURI which is why we want a null if nothing can be found.